partials/nav-main.php

```diff
@@ -1,55 +1,79 @@
 <?php
-
+/**
+ * Main Navigation
+ *
+ * This is the primary navigation for "verticals" on a homepage. Other pages use 
+ * sticky navigation if enabled.
+ *
+ * @link URL
+ *
+ * @package Largo
+ *
+ * If is home page OR sticky nav is disabled show this Main Navigation.
+ */
 if ( is_front_page() || is_home() || !of_get_option( 'show_sticky_nav' ) ): ?>
```

Comment:
@aschweigert this convenience function is now in #916. I'm going to close this pull and reopen submitting to develop. Once #916 is in the replacement pull for this and additionally #912 can both adopt the convenience function.